packages/documentation/src/stories/components/textarea/textarea.stories.tsx

```diff
@@ -0,0 +1,300 @@
+      },
+      table: {
+        category: 'States'
+      }
+    },
+    validation: {
+      name: 'Validation',
+      description: 'Controls the validation state appearance of the component.',
+      control: {
+        type: 'select',
+        labels: {
+          'null': 'Default',
+          'is-valid': 'Valid',
+          'is-invalid': 'Invalid'
+        }
+      },
+      options: [
+        'null',
+        'is-valid',
+        'is-invalid'
+      ],
+      table: {
+        category: 'States'
+      }
+    },
+    validFeedback: {
+      name: 'Valid Feedback',
+      description: 'Text to show when the component is in an valid state.',
+      control: {
+        type: 'text'
+      },
+      table: {
+        category: 'States'
+      }
+    },
+    invalidFeedback: {
+      name: 'Invalid Feedback',
+      description: 'Text to show when the component is in an invalid state.',
+      control: {
+        type: 'text'
+      },
+      table: {
+        category: 'States'
+      }
+    }
+  }
+} as Meta;
+
+const Template = (args: Args, story: Story) => {
+  const id = `ExampleTextarea_${story.name}`;
+  const classes = [
+    'form-control',
+    args.size,
+    args.validation
+  ].filter(c => c && c !== 'null').join(' ');
+
+  const component = <textarea
+    id={ id }
+    className={ classes }
+    placeholder=" "
+    rows={ args.rows }
+    disabled={ args.disabled }
+    aria-label={ args.label }
+    aria-invalid={ VALIDATION_STATE_MAP[args.validation] }
+  ></textarea>;
+
+  const contextuals: (JSX.Element | null)[] = [
+    args.validation === 'is-valid' ? <p key="valid" className="valid-feedback">{ args.validFeedback }</p> : null,
+    args.validation === 'is-invalid' ? <p key="invalid" className="invalid-feedback">{ args.invalidFeedback }</p> : null,
+    args.hint !== '' ? <div key="hint" className="form-text">{ args.hint }</div> : null
+  ].filter(f => f !== null);
+
+  if (args.floatingLabel) {
+    return <div className="form-floating">
+      { component }
+      <label htmlFor={ id } className="form-label">{ args.label }</label>
+      { contextuals }
+    </div>;
+  } else { 
+    return [component, contextuals];
+  }
+};
+
+export const Default = Template.bind({}) as Meta;
+
+export const FloatingLabel = Template.bind({}) as Meta;
+FloatingLabel.parameters = {
+  controls: {
+    exclude: [
+      'Size',
+      'Rows',
+      'Helper Text',
+      'Disabled',
+      'Validation',
+      'Valid Feedback',
+      'Invalid Feedback'
+    ]
+  }
+};
+FloatingLabel.args = {
+  floatingLabel: true,
+  hint: ''
+};
+
+export const Size = Template.bind({}) as Meta;
+Size.parameters = {
+  controls: {
+    exclude: [
+      'Label',
+      'Floating Label',
+      'Rows',
+      'Helper Text',
+      'Disabled',
+      'Validation',
+      'Valid Feedback',
+      'Invalid Feedback'
+    ]
+  }
+};
+Size.args = {
+  size: 'form-control-sm',
+  hint: ''
+};
+
+export const Rows = Template.bind({}) as Meta;
```

Comment:
I'm not sure this story adds a lot of value. There is nothing special to keep in mind and no addidtional docs. In my opinion, the default story captures this use-case. The usage can be described in the args table.